rwengine/src/data/AnimGroup.cpp

```diff
@@ -179,6 +179,162 @@ AnimGroup AnimGroup::getBuiltInAnimGroup(AnimationSet &animations,
         {"phone_talk"},
     }};
 
+    if (name == "player") {
+        group.animations_[0].name = "walk_player";
+        group.animations_[1].name = "run_player";
+        group.animations_[2].name = "sprint_civi";
+        group.animations_[3].name = "idle_stance";
+        group.animations_[4].name = "walk_start";
+    }
```

Comment:
nit: I feel these could be done differently. Otherwise one might accidentally use a wrong index somewhere and it will screw everything up.